### DISCUSSION TITLE
Can multiple Harvesters have multiple unloaders working?

### DISCUSSION THREAD

**Discussion by mountainboomer** [Courseplay/Courseplay_FS22#3207] [Q&A]
If I have a very large field and I am using 3 harvesters, can one unloader handle all three or can you run more than one unloader.  I don't see a way to pair up an unloader with a harvester.  If there are 3 unloaders on the field how is it determined which one responds?

**Comment by Tensuko** [comment]
Just add more unloaders and they will figure out wich unloader goes to wich harvester.
